chainercv/links/model/fpn/faster_rcnn.py

```diff
@@ -45,18 +48,36 @@ class FasterRCNN(chainer.Chain):
 
     """
 
-    _stride = 32
+    stride = 32
```

Comment:
Why did you make this member public?

Reply:
This value is necessary for Transform.

Reply:
It was not possible to reuse method of `FasterRCNN` inside `Transform`.
I had to expose these attributes.